addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,399 @@
+account      / account.reconcile.model.template / match_total_amount (boolean)  : NEW 
+account      / account.reconcile.model.template / match_total_amount_param (float): NEW 
+account      / account.reconcile.model.template / rule_type (selection)         : NEW required: required, selection_keys: ['invoice_matching', 'writeoff_button', 'writeoff_suggestion'], req_default: function
+# NOTHING TO DO: new features
+
+account      / account.tax              / cash_basis_account (many2one) : was renamed to cash_basis_account_id [nothing to do]
+account      / account.tax.template     / cash_basis_account (many2one) : was renamed to cash_basis_account_id [nothing to do]
+# DONE: pre-migration: renamed fields
+
+account      / account.tax              / cash_basis_base_account_id (many2one): previously in module account_cash_basis_base_account
+account      / account.tax.template     / cash_basis_base_account_id (many2one): NEW relation: account.account.template
+account      / account.tax.template     / company_id (many2one)         : DEL relation: res.company
+# NOTHING TO DO
+
+account      / account.tax              / tax_adjustment (boolean)      : DEL
+account      / account.tax              / type_tax_use (selection)      : selection_keys is now '['adjustment', 'none', 'purchase', 'sale']' ('['none', 'purchase', 'sale']')
+account      / account.tax.template     / tax_adjustment (boolean)      : DEL
+account      / account.tax.template     / type_tax_use (selection)      : selection_keys is now '['adjustment', 'none', 'purchase', 'sale']' ('['none', 'purchase', 'sale']')
+# DONE: post-migration: added the boolean field as a new key in the selection
+
+account      / digest.digest            / kpi_account_total_revenue (boolean): NEW
+# NOTHING TO DO
+
+account      / res.company              / account_bank_reconciliation_start (date): NEW
+account      / res.company              / account_dashboard_onboarding_state (selection): NEW selection_keys: ['closed', 'done', 'just_done', 'not_done']
+account      / res.company              / account_invoice_onboarding_state (selection): NEW selection_keys: ['closed', 'done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_invoice_layout_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_sale_tax_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_sample_invoice_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+# DONE: post-migration: marked this annoying helpers as done because a migrated DB is more than already on-boarded
+
+account      / res.company              / account_purchase_tax_id (many2one): NEW relation: account.tax
+account      / res.company              / account_sale_tax_id (many2one): NEW relation: account.tax
+??
```

Comment:
@pedrobaeza what do you think of this?

Reply:
Good question. This passed unadvertised in my review. We have to get it from the ir.property that was set in previous versions.

Reply:
Which properties? :S

Reply:
Done in 8cce611